cairo-contracts/Scarb.toml

```diff
@@ -5,6 +5,9 @@ members = [
   "packages/contracts",
   "packages/core",
   "packages/utils",
+  "cairo-libs/protobuf",
+  "cairo-libs/cometbft",
+  "cometbft",
```

Comment:
Can `cairo-libs` be having it own Scarb workspace, not mixed here?

Reply:
Ah. I tried not to separate the workspace - as we still need to sync the `cairo_test` and `snforge_std` among each other.

Reply:
But I just noticed, the deps are at `dev-deps` - so it's alright. I will move to a separate workspace at project root.

Reply:
89a8fad